app.js

```diff
@@ -52,3 +52,23 @@ function fakeFind(array, callback) {
   }
   return undefined;
 }
+////forEach
+
+
+function fakeForEach(array, callback) {   
+    for(i=0;i<array.length;i++){
+      if (callback(array[i])) {
+        return true;
+      }
+    }
+}
+//fakeIncludes
```

Comment:
Acá también, funciona perfecto. Sólo cambiaría el for por for of y agregaría el let para definir exist y algún punto y coma :)